CMP1903_A1_2324/Game.cs

```diff
@@ -8,14 +8,36 @@ namespace CMP1903_A1_2324
 {
     internal class Game
     {
-        /*
-         * The Game class should create three die objects, roll them, sum and report the total of the three dice rolls.
-         *
-         * EXTRA: For extra requirements (these aren't required though), the dice rolls could be managed so that the
-         * rolls could be continous, and the totals and other statistics could be summarised for example.
-         */
-
-        //Methods
-
+        // Method generates 3 dice rolls and sums them
+        public int RollDice()
+        {   
+            // Create random object
+            Random random = new Random();
+            
+            // Create 3 dice objects
+            Die die1 = new Die();
+            Die die2 = new Die();
+            Die die3 = new Die();
+            
+            // Roll the dices
+            var roll1 = die1.Roll(random);
```

Comment:
You alternate between explicitly-typed and implicitly-typed variables (var and Die), try sticking to one.